src/Equinor.ProCoSys.IPO.WebApi/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:5.0.102-ca-patch-buster-slim AS build
```

Comment:
Needed in pres + library too?

Reply:
If building before a permanent fix is released - yes.